### DISCUSSION TITLE
Illegal instruction 4

### DISCUSSION THREAD

**Discussion by rytash** [simnibs/simnibs#262] [8. Simulation workflows]
Hi All,
I am running charm on my T1 and T2 head scans. I have been running these without incident for 5 scans. Now I am seeing this new error: 
line 2: 72752 Illegal instruction: 4  "/Users/ashlab/Applications/SimNIBS-4.0/simnibs_env/bin/python" -E -u "/Users/ashlab/Applications/SimNIBS-4.0/simnibs_env/lib/python3.9/site-packages/simnibs/cli/charm.py" "$@"

Web searching Illegal instruction 4, it says this happens when there is code that is not runnable with the current OS. I wonder if anyone has seen this before or has any suggestions on how to troubleshoot it?
Thanks a ton!

**Comment by Kstoffer** [comment]
It is difficult to understand the background of this error without more information and debugging. Would you mind trying to install the latest prerealease (4.1a at https://github.com/simnibs/simnibs/releases/tag/dev)? Otherwise we would need more information like the exact version of the OS, platform etc.

**Comment by rytash** [comment]
Thank you Kstoffer, I see. Yes I will try installing version 4.1a of
simnibs, and see if I can reproduce the bug
